.github/workflows/labeler.yml

```
@@ -0,0 +1,56 @@
+name: Smoke check
+run-name: Label - ${{ format('PR {0}', github.event.pull_request.number)
```

actionlint

unexpected EOF while lexing expression

(expression)